sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -310,6 +310,10 @@ case class Distinct(child: LogicalPlan) extends UnaryNode {
   override def output: Seq[Attribute] = child.output
 }
 
+case class Coalesce(numPartitions: Int, shuffle: Boolean, child: LogicalPlan) extends UnaryNode {
```

Comment:
Apparently Repartition also exists :( I'm reverting to CoalescePartitions.
On Apr 28, 2015 6:29 PM, "Reynold Xin" <notifications@github.com> wrote:

> In
> sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala
> <https://github.com/apache/spark/pull/5762#discussion_r29304752>:
>
> > @@ -310,6 +310,10 @@ case class Distinct(child: LogicalPlan) extends UnaryNode {
> >    override def output: Seq[Attribute] = child.output
> >  }
> >
> > +case class Coalesce(numPartitions: Int, shuffle: Boolean, child: LogicalPlan) extends UnaryNode {
>
> yup repartition ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/5762/files#r29304752>.
>